src/api/Comments.ts

```diff
@@ -0,0 +1,34 @@
+import { BASE_URL } from './api';
+
+export const getPostComments = async (id: number) => {
+  const response = await fetch(`${BASE_URL}/comments`);
+
+  const comments = await response.json();
+
+  return comments.filter((comment: CommentInfo) => comment.postId === id);
```

Comment:
use the API ability 
<img width="495" alt="image" src="https://user-images.githubusercontent.com/64437769/154471226-37d49830-51cd-46ea-b269-7486b6b8f70c.png">